File: angular/src/models/address.ts
export class Address {
    AddressId: number;
    StreetAddress: string;
    City: string;
    State: string;
    ZipCode: string;
    constructor(id: number, addr: string, city: string, state: string, zip: string) {
        this.AddressId = id;
        this.StreetAddress = addr;
        this.City = city;
        this.State = state;
        this.ZipCode = zip;
    }
}

File: address/src/Revature.Address.DataAccess/Entities/Address.cs
using System;
using System.ComponentModel.DataAnnotations;


namespace Revature.Address.DataAccess.Entities
{

  /// <summary>
  /// This is an address object for data access purposes,
  /// specifies a guid id for the primary key, as well as a
  /// street, city, state, country, and zip code
  /// </summary>
  public partial class Address
  {
    [Key]
    public Guid? Id { get; set; }
    [Required]
    public string Street { get; set; }
    [Required]
    public string City { get; set; }
    [Required]
    public string State { get; set; }
    [Required]
    public string Country { get; set; }
    [Required]
    public string ZipCode { get; set; }
  }
}
